joomla/joomla-cms
# [4] Move check for offline mode from render to dispatch so that modules are not rendered on 403

Closes https://github.com/joomla/joomla-cms/issues/29455

### Summary of Changes

Move check for offline mode from `render` to `dispatch` method so that modules are not rendered on 403

### Testing Instructions

Set your Joomla 4 site to offline mode in Joomla global config

Visit the frontend. 

Attempt to login as admin - you can - great - you can see the site - great. 

Attempt to go to https://example.com/index.php/component/config = 403 Error (Some module positions will render like headers and footers and maybe menus depending on your template 

Apply PR, repeat test, this time at https://example.com/index.php/component/config you will get a login box. 

### Actual result BEFORE applying this Pull Request

Attempt to go to https://example.com/index.php/component/config = 403 Error (Some module positions will render like headers and footers and maybe menus depending on your template 

### Expected result AFTER applying this Pull Request

this time at https://example.com/index.php/component/config you will get a login box. You can login to see your site

### Also

Cleans up two unused language vars.

### Documentation Changes Required

None
